src/components/SiteHeader.js

```diff
@@ -73,24 +85,49 @@ const SiteHeader = () => {
                 url: "/data/explorer",
                 icon: <BarChartOutlined />,
                 text: "Explorer",
-                disabled: !isAuthenticated,
+                disabled: !canQueryData,
                 key: "explorer",
             },
-            // TODO: Only show if admin / can data manage anything
             {
-                key: "data-manager",
-                url: "/data/manager",
-                icon: <FolderOpenOutlined />,
-                text: "Data Manager",
-                disabled: !isAuthenticated,
-            },
-            {
-                key: "services",
-                url: "/services",
-                icon: <DashboardOutlined />,
-                text: "Services",
-                disabled: !isAuthenticated,
+                url: "/genomes",
+                icon: <FileTextOutlined />,
+                text: "Reference Genomes",
+                key: "genomes",
             },
+            ...(managerPermissions.canManageAnything ? [
+                {
+                    key: "data-manager",
+                    url: "/data/manager",
+                    icon: <FolderOpenOutlined />,
+                    text: "Data Manager",
+                    disabled: !isAuthenticated || !managerPermissions.canManageAnything,
+                },
+                // For now, only show the services page to users who can manage something, since it's not useful for
+                // end users.
+                {
+                    key: "services",
+                    url: "/services",
+                    icon: <DashboardOutlined />,
+                    text: "Services",
+                    disabled: !isAuthenticated,
```

Comment:
same reasoning here as above, will never be `disabled` since this menu item only appears conditionally